README.md

```diff
@@ -1,18 +1,18 @@
-# ThingsBoard MQTT client Python SDK
+# ThingsBoard MQTT and HTTP client Python SDK
 [![Join the chat at https://gitter.im/thingsboard/chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/thingsboard/chat?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 <a href="https://thingsboard.io"><img src="./logo.png?raw=true" width="100" height="100"></a>
 
 ThingsBoard is an open-source IoT platform for data collection, processing, visualization, and device management.
-This project ia a Python library that provides convenient client SDK for both [Device](https://thingsboard.io/docs/reference/mqtt-api/) 
-and [Gateway](https://thingsboard.io/docs/reference/gateway-mqtt-api/) APIs.
+This project ia a Python library that provides convenient client SDK for both Device and [Gateway](https://thingsboard.io/docs/reference/gateway-mqtt-api/) APIs.
 
 SDK supports:
-- Unencrypted and encrypted (TLS v1.2) connection;
-- QoS 0 and 1;
-- Automatic reconnect;
-- All Device MQTT APIs provided by ThingsBoard
-- All Gateway MQTT APIs provided by ThingsBoard
+- Unencrypted and encrypted (TLS v1.2) connection
+- QoS 0 and 1
+- Automatic reconnect
+- All [Device MQTT](https://thingsboard.io/docs/reference/mqtt-api/) APIs provided by ThingsBoard
+- All [Gateway MQTT](https://thingsboard.io/docs/reference/gateway-mqtt-api/) APIs provided by ThingsBoard
+- All [Device HTTP](https://thingsboard.io/docs/reference/http-api/) APIs provided by ThingsBoard
 
 SDK is based on Paho MQTT library. 
```

Comment:
Should we write here something about the requests library?

Reply:
I've documented the missing features and mentioned the Requests library. I will add the missing features with a PR later.